modules/express/README.md

```diff
@@ -46,6 +46,35 @@ Environment: test
 Base URI: http://0.0.0.0:4000
 ```
 
+## Docker in Production
```

Comment:
Use `###`, I think this should be at the same level as "Building the docker container". Also I think this would make more sense below that section.